src/main.ts

```diff
@@ -49,24 +50,50 @@ async function run() {
       throw error;
     }
 
-    const event = status > 0 ? "REQUEST_CHANGES" : "COMMENT";
+    const failed = status > 0;// ? "REQUEST_CHANGES" : "COMMENT";
+    if (failed) {
+      setFailed("Failed");
+    }
     const body = [
-      `## [size-limit](${SIZE_LIMIT_URL}) report`,
+      SIZE_LIMIT_HEADING,
       table(limit.formatResults(base, current))
     ].join("\r\n");
 
-    try {
-      octokit.pulls.createReview({
-        ...repo,
-        // eslint-disable-next-line camelcase
-        pull_number: pr.number,
-        event,
-        body
-      });
-    } catch (error) {
-      console.log(
-        "Error creating PR review. This can happen for PR's originating from a fork without write permissions."
-      );
+    const { data: commentList } = await octokit.issues.listComments({
+      ...repo,
+      issue_number: pr.number
+    });
+
+    const sizeLimitComment = commentList.find(comment =>
+      comment.body.startsWith(SIZE_LIMIT_HEADING)
+    );
+
+    if (sizeLimitComment == undefined) {
+      try {
+        octokit.issues.createComment({
+          ...repo,
+          // eslint-disable-next-line camelcase
+          issue_number: pr.number,
+          body
+        });
+      } catch (error) {
+        console.log(
+          "Error creating comment. This can happen for PR's originating from a fork without write permissions."
+        );
+      }
+    } else {
+      try {
+        octokit.issues.updateComment({
```

Comment:
This was a bug in #18 that I fixed in #20 - it can result in an unhandled promise rejection (which occurred in the test. Needs an `await``octokit.issues.createComment` does as well.